From 622360c2c4335fff793f81f5e673326a581f5c0c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Thu, 19 Apr 2018 18:25:07 +0300
Subject: [PATCH] aom: Remove unsupported pixel formats

Support for these was apparently never in the codebase, and the enum
values were recently removed from the public headers [1].

This fixes build with latest libaom build.

[1] https://aomedia.googlesource.com/aom/+/3f29cc20e3a4c348cb41a797c68de856ddb84e12
---
 modules/codec/aom.c | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/modules/codec/aom.c b/modules/codec/aom.c
index 5bb442b5d50..60034158c47 100644
--- a/modules/codec/aom.c
+++ b/modules/codec/aom.c
@@ -116,17 +116,6 @@ static const struct
 
     { VLC_CODEC_YV12, AOM_IMG_FMT_YV12, 8, 0 },
     { VLC_CODEC_YUVA, AOM_IMG_FMT_444A, 8, 0 },
-    { VLC_CODEC_YUYV, AOM_IMG_FMT_YUY2, 8, 0 },
-    { VLC_CODEC_UYVY, AOM_IMG_FMT_UYVY, 8, 0 },
-    { VLC_CODEC_YVYU, AOM_IMG_FMT_YVYU, 8, 0 },
-
-    { VLC_CODEC_RGB15, AOM_IMG_FMT_RGB555, 8, 0 },
-    { VLC_CODEC_RGB16, AOM_IMG_FMT_RGB565, 8, 0 },
-    { VLC_CODEC_RGB24, AOM_IMG_FMT_RGB24, 8, 0 },
-    { VLC_CODEC_RGB32, AOM_IMG_FMT_RGB32, 8, 0 },
-
-    { VLC_CODEC_ARGB, AOM_IMG_FMT_ARGB, 8, 0 },
-    { VLC_CODEC_BGRA, AOM_IMG_FMT_ARGB_LE, 8, 0 },
 
     { VLC_CODEC_GBR_PLANAR, AOM_IMG_FMT_I444, 8, 1 },
     { VLC_CODEC_GBR_PLANAR_10L, AOM_IMG_FMT_I44416, 10, 1 },
